lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_new_milestone_action.rb

```diff
@@ -48,6 +50,18 @@ def self.available_options
                                        optional: true,
                                        is_string: false,
                                        default_value: false),
+          FastlaneCore::ConfigItem.new(key: :milestone_duration,
+                                       env_name: 'GHHELPER_MILESTONE_DURATION',
```

Comment:
IMHO we don't need an `env_name` for this, as it IMHO encourages magic global variables set in the `Fastfile`. 

Replacing that with a constant in the `Fastfile` that's passed as an argument to this method is IMHO far better, even if it kinda looks the same code-wise – it'll hopefully push us toward better (and more testable) patterns.

Reply:
If it was up to me, I'd burn all of the ENV vars with eternal fire as it's nothing but global variables on steroids. I was only following the code conventions we currently have in the repository, but might have misinterpreted it.

The PR is auto-merged and I don't think it's worth following up right now. Let me know if you disagree and I'll open a new PR.